docs/server-side/v2/introduction/webhook.mdx

```diff
@@ -1086,6 +1097,15 @@ This event will be sent when there are failures in HLS streaming and/or recordin
 1. The meeting_url is invalid
 2. Other unexpected errors with HLS streaming / recording.
 
```

Comment:
@chdinesh1089 let's include the message here as well:

Please check the below table for possible error types, messages and description.